i18n/language/pluralspec_test.go

```diff
@@ -504,6 +504,25 @@ func TestUkrainian(t *testing.T) {
 	runTests(t, "uk", tests)
 }
 
+func TestThai(t *testing.T) {
+	tests := []pluralTest{
```

Comment:
There are already languages that always return Other, so these tests should be identical to those (e.g. Chinese).

Perhaps it would be good to make a helper test function that tests languages which always return Other.